tests/playwright/src/specs/podman-machine-resources.spec.ts

```diff
@@ -0,0 +1,208 @@
+          page,
+          RESOURCE_NAME,
+          DEFAULT_PODMAN_MACHINE_VISIBLE,
+        );
+
+        await playExpect(resourcesPodmanConnections.resourceElementDetailsButton).toBeVisible({ timeout: 10_000 });
+        await resourcesPodmanConnections.resourceElementDetailsButton.click();
+
+        const podmanMachineDetails = new PodmanMachineDetails(page, DEFAULT_PODMAN_MACHINE);
+
+        await playExpect(podmanMachineDetails.podmanMachineStatus).toHaveText('RUNNING', { timeout: 60_000 });
+        await playExpect(podmanMachineDetails.podmanMachineStopButton).toBeEnabled();
+        await podmanMachineDetails.podmanMachineStopButton.click();
+        await playExpect(podmanMachineDetails.podmanMachineStatus).toHaveText('OFF', { timeout: 60_000 });
+      });
+
+      test('Create a machine through Resources page', async ({ page, navigationBar }) => {
+        test.setTimeout(200_000);
+
+        const settingsBar = await navigationBar.openSettings();
+        await settingsBar.resourcesTab.click();
+
+        const podmanResources = new ResourceConnectionCardPage(page, 'podman');
+        await podmanResources.createButton.click();
+
+        const createMachinePage = new CreateMachinePage(page);
+        const resourcePage = await createMachinePage.createMachine(PODMAN_MACHINE_NAME, {
+          isRootful: isRoot,
+          enableUserNet: userNet,
+          setAsDefault: false,
+          startNow: false,
+        });
+
+        await playExpect(resourcePage.heading).toBeVisible();
+        const machineBox = new ResourceConnectionCardPage(page, 'podman', PODMAN_MACHINE_NAME); //does not work with visible name
+
+        await playExpect(machineBox.resourceElementDetailsButton).toBeVisible({ timeout: 30_000 });
+        await machineBox.resourceElementDetailsButton.click();
+
+        const podmanMachineDetails = new PodmanMachineDetails(page, MACHINE_VISIBLE_NAME);
+        await playExpect(podmanMachineDetails.podmanMachineName).toBeVisible({ timeout: 30_000 });
+        await playExpect(podmanMachineDetails.podmanMachineStatus).toHaveText('OFF');
+
+        await playExpect(podmanMachineDetails.podmanMachineStartButton).toBeEnabled();
+        await podmanMachineDetails.podmanMachineStartButton.click();
+
+        await playExpect(dialog).toBeVisible({ timeout: 60_000 });
+        await handleConfirmationDialog(page, 'Podman', true, 'Yes');
+        await handleConfirmationDialog(page, 'Podman', true, 'OK');
+
+        await playExpect(podmanMachineDetails.podmanMachineStatus).toHaveText('RUNNING', { timeout: 90_000 });
+
+        await playExpect(podmanMachineDetails.podmanMachineStopButton).toBeEnabled();
+        await podmanMachineDetails.podmanMachineStopButton.click();
+        await playExpect(podmanMachineDetails.podmanMachineStatus).toHaveText('OFF', { timeout: 60_000 });
+      });
+
+      test('Restart default podman machine', async ({ page, navigationBar }) => {
```

Comment:
Also, as per our discussion, restart the machine we work with in parametrized tests, not the default one since the functionality is covered in other test.